docs/book/advanced-guide/getting_started_on_gcp.md

```diff
@@ -0,0 +1,591 @@
+    --member="user:"${USER_EMAIL} --role="roles/iam.serviceAccountUser"
+```
+{% endtab %}
+{% endtabs %}
+
+## ZenML Stack
+
+Everything on the GCP side is set up, you're ready to set up the ZenML stack 
+components now. 
+
+Copy-paste this into your terminal and press enter.
+
+```bash
+zenml orchestrator register vertex_orchestrator --flavor=vertex \
+      --project=$PROJECT_NUMBER --location=$GCP_LOCATION \
+      --workload_service_account=$SERVICE_ACCOUNT
+zenml secrets-manager register gcp_secrets_manager \
+      --flavor=gcp_secrets_manager --project_id=$PROJECT_NUMBER
+zenml container-registry register gcp_registry --flavor=gcp \
+      --uri=$CONTAINER_REGISTRY_URI
+zenml artifact-store register gcp_artifact_store --flavor=gcp \
+      --path=$GSUTIL_URI
+zenml metadata-store register gcp_metadata_store --flavor=mysql \
+      --host=$DB_HOST --port=3306 --database=$DB_NAME \
+      --secret=mysql_secret
+zenml stack register gcp_vertex_stack -m gcp_metadata_store \
+      -a gcp_artifact_store -o vertex_orchestrator -c gcp_registry \
+      -x gcp_secrets_manager --set
+zenml secret register mysql_secret --schema=mysql \
+      --user=root --password=$DB_PASSWORD \
+      --ssl_ca="@"$SERVER_CERT_PATH --ssl_cert="@"$CLIENT_CERT_PATH \
+      --ssl_key="@"$CLIENT_KEY_PATH
+```
+
+This is where your ZenML stack is created and connected to the GCP cloud 
+resources. If you now run `zenml stack describe` you should see this:
+
+```bash
+            Stack Configuration             
+┏━━━━━━━━━━━━━━━━━━━━┯━━━━━━━━━━━━━━━━━━━━━┓
+┃ COMPONENT_TYPE     │ COMPONENT_NAME      ┃
+┠────────────────────┼─────────────────────┨
+┃ ARTIFACT_STORE     │ gcp_artifact_store  ┃
+┠────────────────────┼─────────────────────┨
+┃ CONTAINER_REGISTRY │ gcp_registry        ┃
+┠────────────────────┼─────────────────────┨
+┃ METADATA_STORE     │ gcp_metadata_store  ┃
+┠────────────────────┼─────────────────────┨
+┃ ORCHESTRATOR       │ vertex_orchestrator ┃
+┠────────────────────┼─────────────────────┨
+┃ SECRETS_MANAGER    │ gcp_secrets_manager ┃
+┗━━━━━━━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━┛
+     'gcp_vertex_stack' stack (ACTIVE)  
+```
+
+## Run your pipeline in the cloud
+
+With your ZenML stack set up and active, you are now ready to run your ZenML
+pipeline on Vertex AI. 
+
+For example, you could pull the ZenML Vertex AI example and run it.
+
+```bash
+zenml example pull vertex_ai_orchestration
+cd zenml_examples/vertex_ai_orchestration/
+python run.py
+```
+
+{% hint style="info" %}
+Your first run might fail as one of the service accounts is only created once 
+vertex is run for the first time. This service account will need to be given
+appropriate rights after the first run fails.
+{% endhint %}
+
+At the end of the logs you should be seeing a link to the Vertex AI dashboard. 
+It should look something like this:
+
+![Finished Run](../assets/VertexAiRun.png)
+
+In case you get an error message like this:
```

Comment:
Is there a possibility that this error doesn't occur for a new user? If not, then we can directly include the following instructions as part of creating the service account as a subsection to avoid the chance of running into this error in the first place.

Reply:
Oh I see, the service account isn't available before the first run. That is super weird